### DISCUSSION TITLE
[BUG] migrate express 相关的文档打不开了

### DISCUSSION THREAD

**Discussion by skypesky** [midwayjs/midway#3965] [Q&A] [answered]
### Describe the problem（描述问题）

https://github.com/user-attachments/assets/45aa4092-48a5-462a-8782-ae0c77a0ff05

### Midway Versions（Midway 版本）

不适用

**Comment by czy88840616** [comment, marked as answer]
这个是三方服务索引的问题，我也控制不了-。-

**Reply by czy88840616** [reply]
删了下索引，应该可以了
